### ISSUE TITLE
Can`t predict the correct categories.

### ISSUE THREAD

**Issue by DeepStillWater** [fizyr/keras-retinanet#917]
Hello,I am sorry to disturb you again, I follow the steps to train the model,and convert it to the right inference model,I can`t get the right result. I just modify the categories to two classes, and the training  and testing result are showing as:

 Training result:
...
9988/10000 [============================>.] - ETA: 5s - loss: 4.1035 - regressi 
9989/10000 [============================>.] - ETA: 5s - loss: 4.1035 - regressi 
9990/10000 [============================>.] - ETA: 4s - loss: 4.1035 - regressi 
9991/10000 [============================>.] - ETA: 4s - loss: 4.1035 - regressi 
9992/10000 [============================>.] - ETA: 3s - loss: 4.1035 - regressi 
9993/10000 [============================>.] - ETA: 3s - loss: 4.1035 - regressi 
9994/10000 [============================>.] - ETA: 2s - loss: 4.1035 - regressi 
9995/10000 [============================>.] - ETA: 2s - loss: 4.1035 - regressi 
9996/10000 [============================>.] - ETA: 1s - loss: 4.1035 - regressi 
9997/10000 [============================>.] - ETA: 1s - loss: 4.1035 - regressi 
9998/10000 [============================>.] - ETA: 0s - loss: 4.1035 - regressi 
9999/10000 [============================>.] - ETA: 0s - loss: 4.1035 - regressi
10000/10000 [==============================] - 4833s 483ms/step - loss: 4.1035 - regression_loss: 2.9765 - classification_loss: 1.1270

Testing result:

(1, 800, 1066, 3) //image shapes
processing time:  0.08204317092895508
box: [-2.5 -2.5 -2.5 -2.5]
score: -1.0
label: -1

**Comment by hgaiser** [comment]
Most likely your training data is incorrect. Try using debug.py.

Since this is not an issue with our code, I'm going to close this.

**Comment by Vietdung113** [comment]
i got same problem, i was run debug.py to see my data. and that seem correct, all box is green, i got high MAP, but when i test on new image is still have problem, didn't u sloved it?
